### ISSUE TITLE
[WASM] UNO 5 no longer builds

### ISSUE THREAD

**Issue by DierkDroth** [unoplatform/uno#14242]
### Current behavior

Couple of issues:
- I found that my GitHub Actions YAML no longer works (screenshots below)
- Next I tried to build UNO WASM locally on my PC which fails as well (screenshot below)
- lastly Acronis Cyber Protect (running on my local PC) puts mono to quarantine (screenshot below)

At this point I'm dead in the water. What to do to get me going again?
<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/a2ddd0db-9d90-43c7-aa01-768ef3a9509e">
<img width="912" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/add9faa6-1911-48de-a2b0-50d53e3547c5">
<img width="867" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/579c236c-1dc0-4a35-b3f6-db91cc17438d">
<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/ee8e643c-7368-4e08-98ea-02c62fc3b237">

### Works on UWP/WinUI

None

**Comment by Youssef1313** [comment]
Can you attach a binlog?

**Comment by DierkDroth** [comment]
Sure

[msbuild.zip](https://github.com/unoplatform/uno/files/13237596/msbuild.zip)

**Comment by Youssef1313** [comment]
@DierkDroth Does downgrading wasm bootstrapper to `7.0.31` work? Maybe there is something going on when using 8.0 bootstrapper while building for net7.0?

**Comment by DierkDroth** [comment]
@Youssef1313 thanks for looking into.

Please be specific on what you wanted me to downgrade: just 'Uno.Wasm.Bootstrap' or 'Uno.Wasm.Bootstrap.DevServer' as well (screenshot below)?

<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/c3c355c2-2a08-4eef-91a1-693b57d0935c">

**Comment by Youssef1313** [comment]
Both Uno.Wasm.Bootstrap and Uno.Wasm.Bootstrap.DevServer. Note that this isn't saying this is the fix, this is just to understand if it's a viable workaround and/or if it's related to using .NET 7 with 8.x bootstrapper.

**Comment by DierkDroth** [comment]
@Youssef1313 I confirm that downgrading to 7.0.31 fixes the issue for local and GitHub actions builds.

Note: I have latest official .NET 7 and VS installed. I won't touch any .NET 8 preview nor VS preview with any 5 foot stick but rather wait until 14th and retry the officials releases.

**Comment by jeromelaban** [comment]
Thanks for the report. The build is failing because the mono aot compiler is quarantined by your anti-virus, there's nothing we can do here. It's a false positive, particularly if it's marked as "malware.generic".

You can either mark the binary a safe, or downgrade to another version of the uno.wasm.bootstrap packages.

**Comment by DierkDroth** [comment]
>> The build is failing because the mono aot compiler is quarantined by your anti-virus
@jeromelaban this actually is not correct, since the the failed GitHub actions build has nothing to do with my local AntiVirus SW. As per my reports above using `Uno.Wasm.Bootstrap(.DevServer) 8.X` just does not work with .NET 7.

**Comment by jeromelaban** [comment]
Thanks for the update. The statement above is correct. If it fails on github, it's for a different reason. You can troubleshoot by [using these steps](https://platform.uno/docs/articles/uno-builds-troubleshooting.html).

**Comment by DierkDroth** [comment]
@jeromelaban did you see the binlog which I created and attached already this AM by follwing the link you provided?

**Comment by jeromelaban** [comment]
@DierkDroth yes Idid, those tell that the mono-aot-cross.exe file does not exist, which is consistent with the anti-virus behavior. Please the binlogs from GitHub for clues.

**Comment by DierkDroth** [comment]
@jeromelaban hmm ... I never have tried to create a binlog on GitHub and 'upload' it ... sure I'm aware of the /bl switch which l'll add to the 'Build' action in my YAML file. 

However, would you mind taking a look if the 'Publish Artifact' action in my YAML file above will catch the binlog as well or if I needed to amend the YAML ... and if so, how. Thanks in advance

**Comment by DierkDroth** [comment]
@jeromelaban @Youssef1313 since I don't know how to setup the GitHub YAML to capture and upload the BINLOG on a failed build, I temporarily disabled my Acronis protection system. The build still fails. Here is a new BINLOG

[msbuild.zip](https://github.com/unoplatform/uno/files/13242431/msbuild.zip)

**Comment by Youssef1313** [comment]
I see `[00:05:12.9277939] wasm-ld: error: initial memory too small, 17104928 bytes needed`

Can you try adding `<WasmShellExtraEmccFlags Include="-s INITIAL_MEMORY=32MB" />`?

**Comment by DierkDroth** [comment]
Thanks for looking into @Youssef1313.

To clarify: you mean adding that line here, correct?

<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/5681c6b3-3e58-4a8d-822e-e1a0ae491279">

**Comment by Youssef1313** [comment]
You'll probably need to add it in the unconditional PropertyGroup (after the line `<AllowUnsafeBlocks>True</AllowUnsafeBlocks>`)

**Comment by Youssef1313** [comment]
No, you need to add it to `ItemGroup`.

**Comment by DierkDroth** [comment]
@Youssef1313 thanks, I'll try that

FYI, in fact adding it to any PropertyGroup did invalidate the project file.

**Comment by DierkDroth** [comment]
@Youssef1313 I ran more tests: the parameter you suggested resolved the issue on building on my local machine as well as on GitHub. Thanks

However, now there is the next problem: I found that in the build there now is created a subfolder `wwwroot` which holds important files (like `index.html`) and which was not there before. In fact, all the files and folders in that new `wwwroot` had been one level up before. This new logic screws up my complete build/deply/run logic and makes my product unusable.

How could that unwelcome change be reverted and the `wwwroot `subfolder be eliminated?
